src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,50 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import duke.Deadline;
+import duke.DukeException;
+
+/**
+ * To unit test the Deadline class.
+ */
+public class DeadlineTest {
+
+    /**
+     * Tests the creation of a Deadline object.
+     *
+     * @throws DukeException not thrown.
+     */
+    @Test
+    public void testCreateDeadline() throws DukeException {
```

Comment:
I like how thoroughly you have tested the different classes! Perhaps a minor improvement could be to follow the three part format for test method names where relevant. e.g.  testCreateDeadline_testDeadline_dukeExceptionThrown()